packages/es-components/src/components/containers/tooltip/Tooltip.js

```diff
@@ -197,7 +199,8 @@ Tooltip.propTypes = {
   /** Disables the default show onFocus functionality */
   disableFocus: PropTypes.bool,
   /** Select the color style of the button, types come from theme */
-  styleType: PropTypes.string
+  styleType: PropTypes.string,
+  buttonLabel: PropTypes.string
```

Comment:
I think `buttonLabel` is not quite what we want here I think a better and more flexible approach would be to add something like `linkProps`, which would allow you to do something like:

```jsx
<Tooltip
  linkProps={{ 'aria-label': 'Hello info' }}
  content={<p>World!</p>}
>
  Hello
</Tooltip>
```

```suggestion
  linkProps: PropTypes.object
```